### PR TITLE
Feature/Save and Load

### PR DESCRIPTION
**Pull request by tonyyang-svail**

### CODE REVIEW

File: src/parameter.cc [hunk 1]
```diff
@@ -36,24 +40,75 @@ ParameterHandle ParameterCollection::AddParameter(
...
+  if (directory_name.empty()) {
+    auto t = std::time(nullptr);
+    auto tm = *std::localtime(&t);
+    std::stringstream ss;
+    ss << std::put_time(&tm, "%d-%m-%Y %H-%M-%S");
+    directory_name = "/tmp/" + ss.str() + "/";
+  }
+  VLOG(3) << directory_name;
+  PADDLE_ENFORCE_EQ(directory_name.back(), '/');
+  PADDLE_ENFORCE_EQ(mkdir(directory_name.c_str(), 0775),
```
Comment (tonyyang-svail) [inline src/parameter.cc]: change to `0644`

Comment (kexinzhao) [reply src/parameter.cc]: What does `0644` or `0775` mean here?

Comment (tonyyang-svail) [reply src/parameter.cc]: permission `int mkdir(const char *path, mode_t mode);`
1. `0644` means `rw-rw-r--`
1. `0775` means `rwxrwxr-x`

**Review by kexinzhao — APPROVED** [review]
Excellent!

**Pull request merged**